quickstarts/microsoft.digitaltwins/digitaltwins-with-function-time-series-database-connection/main.bicep

```diff
@@ -0,0 +1,126 @@
+// New Resources Name
+@description('Name given to Digital Twins resource')
+param digitalTwinsName string
```

Comment:
add a globally unique defaultValue (here and where possible), makes it must easier to deploy when kicking the tires...

Reply:
Changed to add default names